CustomCraftSML/Serialization/Entries/AddedRecipe.cs

```diff
@@ -92,12 +91,26 @@ public override bool SendToSMLHelper()
             }
             catch (Exception ex)
             {
-                QuickLogger.Error($"Exception thrown while handling Added Recipe '{this.ItemID}'{Environment.NewLine}{ex}");
+                QuickLogger.Error($"Exception thrown while handling Added Recipe '{this.ItemID}' from  from {this.Origin}", ex);
```

Comment:
Create a common `LogIdentification` property at the lowest base classes and exposed by the `ICustomCraft` interface.  
This property should return `this.Key`, `this.ItemID`, and `this.Origin` for uniform identification in logs.

Reply:
For a future update